### DISCUSSION TITLE
parallel parsing

### DISCUSSION THREAD

**Discussion by aladdin-add** [kataw/kataw#50] [Ideas]
possible to use node.js worker to parallel parsing?

**Comment by KFlash** [comment]
How do you suggest to implement this?  I think it should work, but maybe better of splitting the CST output into fragments and assemble the tree while done parsing?

**Reply by aladdin-add** [reply]
I'd prefer not put it in the core - just another package on the top of kataw.

**Reply by KFlash** [reply]
Can you test if it works? Auto-error recovery exist, so it should work, but not sure about CST output
